sample/master-detail/README.md

```diff
@@ -1,5 +1,5 @@
 # Sample Master-Detail App
 
-> ⚠  This sample is for advanced single-pane/multi-pane navigation and layout, for generic master-detail navigation please refer to the [Sample Todo List App](https://arkivanov.github.io/Decompose/samples/#sample-master-detail-app).
+> ⚠  This sample is for advanced single-pane/multi-pane navigation and layout, for generic master-detail navigation please refer to the [Sample Todo List App](https://arkivanov.github.io/Decompose/samples/#sample-todo-list-app).
```

Comment:
That's weird, I see two files changed. The master-detail README has a link changed from /#sample-master-detail-app to /#sample-todo-list-app

![image](https://user-images.githubusercontent.com/26204457/151600230-5790e7bb-3138-4b66-ac0c-46109291eb9f.png)

Reply:
Yes , I did change it to todo-list because it belongs to todo app , (once view the whole FILE)
#25  changes this link from todo to master example which is not desired as Master-Detail example has links to both 
 - TODO-APP
 - Master-Detail App itself
![image](https://user-images.githubusercontent.com/44765050/151656386-1a625127-241d-4dfb-8795-b841f0f565c7.png)

Reply:
Yes, you are right. Thanks for your attention!